homeassistant/components/onvif/device.py

```diff
@@ -242,9 +242,13 @@ async def async_get_capabilities(self):
 
         pullpoint = False
         try:
-            event_service = self.device.create_events_service()
-            event_capabilities = await event_service.GetServiceCapabilities()
-            pullpoint = event_capabilities and event_capabilities.WSPullPointSupport
+            self.device.get_definition("pullpoint")
+            await self.device.create_pullpoint_subscription()
+            pullpoint_service = self.device.create_pullpoint_service()
+            await pullpoint_service.PullMessages(
```

Comment:
This is likely unnecessary, and can actually make it so we don't get the initialization messages for the subscription we just created. We shouldn't pull messages unless we are ready to process them. Checking if the subscription was created successfully is probably enough.

Reply:
I add the PullMessages call in order to ensure that the service work correctly. If you think it is not necessary I take it off.